backend/src/main/java/com/project/DataLoader.java

```diff
@@ -35,5 +41,14 @@ public void run(ApplicationArguments args) {
         employeeRepository.save(new Employee("Cory", "Wilson", "Opally", "https://picsum.photos/200/300"));
         employeeRepository.save(new Employee("Vickie", "Harrison", "GrownMan", "https://picsum.photos/200/300"));
         employeeRepository.save(new Employee("Mabel", "Castro", "BatBoy", "https://picsum.photos/200/300"));
+
+
+        Coffee coffee = new Coffee("nowa", "cos tam", "dawda", "dawda", CoffeePreparationDifficulty.EASY, 3);
+        coffee.setPopularity(5);
+        coffeeService.save(coffee);
+        coffee = new Coffee("dawawaw", "cos tam", "dawda", "dawda", CoffeePreparationDifficulty.EASY, 3);
+        coffee.setPopularity(1);
+        coffeeService.save(coffee);
```

Comment:
Lepiej by to było wrzucić do osobnej funkcji, np. CoffeeDataLoader, bo jak więcej osób zacznie tutaj tak dodawać obiekty to się trochę misz masz zrobi 😋